backstop/tests/column-filter.html

```diff
@@ -0,0 +1,223 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html id="theme">
+  <head>
+    <title>Column filter | iTwinUI</title>
+    <link
+      rel="icon"
+      href="../assets/favicon.ico"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/global.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/button.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/inputs.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/toggle-switch.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/table.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      section {
+        padding: 16px;
+      }
+    </style>
+    <script src="../assets/theme.js"></script>
+  </head>
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <h1>Column filter</h1>
+    <hr />
+    <h2 id="default">Text filter</h2>
+
+    <section id="demo-text">
```

Comment:
Without `<section>` that has `fit-content` filter takes whole screen size. Our filter will be placed inside `<div>`, so it will also take whole screen.
Change `display: flex` to `display: inline-flex` in `iui-column-filter`.

Reply:
> @FlyersPh9 Is this toggle really supposed to be on the left? I just look at it and cannot get used to..
> ![image](https://user-images.githubusercontent.com/81580355/116662655-b87c1a80-a99e-11eb-959f-088a5ad65793.png)
> 
> What if we put it aligned all together?
> ![image](https://user-images.githubusercontent.com/81580355/116662708-c893fa00-a99e-11eb-99c5-7c5e71726c37.png)

That's how the standard has it, I believe these were originally laid out by JP.  One thing I thought about proposing was something like the attached Photoshop mockup where the toggle is left aligned but on the same line as the buttons.
![Screen Shot 2021-04-30 at 4 15 40 PM](https://user-images.githubusercontent.com/849817/116749692-2aac2980-a9b6-11eb-99d4-18c0d7b3d526.png)

Reply:
> Change `display: flex` to `display: inline-flex` in `iui-column-filter`.

I don't know why GitHub put that other response in here 😅 But this change is now implemented.